noetic/distribution.yaml

```diff
@@ -1077,6 +1077,22 @@ repositories:
       - cob_teleop
       - generic_throttle
       - scenario_test_tools
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/4am-robotics/cob_command_tools-release.git
+      version: 0.6.35-1
+    source:
+      type: git
+      url: https://github.com/4am-robotics/cob_command_tools.git
+      version: noetic-devel
+    status: maintained
+  cob_command_tools_legacy:
```

Comment:
similar to https://github.com/ros/rosdistro/pull/40670/files#r1570307215

I hope I'm doing the right thing here - could you please confirm that this PR will:
- release a new version `0.6.35` of all packages (except `service_tools`) from `noetic-release-candidate`
- keep the last release `0.6.34` for `service_tools`available based on `indigo_release_candidate`